tests/modules/nf-core/graphtyper/genotype/main.nf

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+params.enable_conda = true
+
+include { GRAPHTYPER_GENOTYPE } from '../../../../../modules/nf-core/graphtyper/genotype/main.nf'
+
+workflow test_graphtyper_genotype {
```

Comment:
Can you also add a test for when there are more than one input BAM files?

Reply:
And can you also add a test with CRAM files? (it's fine if you combine these two suggestions into one test)